### ISSUE TITLE
Strange behavior at Dropdown

### ISSUE THREAD

**Issue by welingtonsampaio** [alboompro/boom-components#15]
I saw a very strange behavior at Dropdown component. The gif show this behavior...

![Screen-Recording-2020-01-21-at-1](https://user-images.githubusercontent.com/1521226/72809080-ad81f880-3c39-11ea-9e77-6fdb7122cd36.gif)

Any idea @elisoncampos ?

**Comment by elisoncampos** [comment]
Yeah, playground from docs acts really strange  with positioning so it worked correctly in playground but not within projects. I'm fixing it right now.

**Comment by elisoncampos** [comment]
Check https://github.com/alboompro/boom-components/commit/c0c8901c14640252da1825d07ee56e8f0f099c9c, fixed it.

https://imgur.com/RshltwQ
